crates/pop-parachains/src/templates.rs

```diff
@@ -79,11 +79,21 @@ pub enum Template {
 		props(Provider = "Pop", Repository = "https://github.com/r0gue-io/base-parachain")
 	)]
 	Base,
+	#[strum(
+		serialize = "nfts",
+		message = "Nfts",
+		detailed_message = "Parachain configured with assets and nfts functionalities.",
+		props(
+			Provider = "Pop",
+			Repository = "https://github.com/r0gue-io/base-parachain/tree/nfts-parachain"
```

Comment:
Suggest a rename to `assets-parachain`

Reply:
Also, as we are presenting releases to install via the CLI, it _might_ be better to separate these into different repos, otherwise we will need to add filtering of releases by tag - think about x templates all needing to be released at the same time based on a Polkadot version update. We will also need to set up additional branch protection rules for each of the templated branches.

Using separate repos means we can use the same release tag versioning, so think we should consider future maintenance requirements before merging this.